src/web/reducers/flows.js

```diff
@@ -334,11 +334,16 @@ reducer = reduceReducers(
               return { ...node, next, lastModified: new Date() }
             })
 
+        const links = (payload.links || currentFlow.links).map(link => ({
+          ...link,
+          points: link.points.map(point => ({ x: Math.round(point.x), y: Math.round(point.y) }))
```

Comment:
NIT, can also be `.map(({ x, y }) => ({ x: Math.round(x), y: Math.round(y) }))`

Reply:
Yep, good point